### PR TITLE
Article large image

### LINKED ISSUE #253: Article - large image
Full edge image, in line. This should go almost entire to the edge, only have 60px buffer on both edges

### PR DESCRIPTION
**Pull request by gestchild**
## What is this PR trying to achieve?

Fixes #253

## What does it look like?

Before:
screen shot 2017-01-16 at 17 01 02

After: 
screen shot 2017-01-16 at 17 00 26

## For interface components
- [x] Browser testing - Have you checked the component in our supported browsers
- [x] No javascript - Have you checked the component with javascript disabled
- [x] A11y testing - Have you run pa11y against the component and fixed/verified all errors, warnings and notices?

### CODE REVIEW

File: client/scss/components/_figure.scss [hunk 1]
```diff
@@ -54,37 +54,49 @@
   }
 }
 
-$large-width: (5/7 * 100)+'%';
+$narrow-width: (5/7) * 100%; // calculates the narrow column width as a percentage of the main column.
+$center-width-large: (6/7) * 100%; // calculates the halfway point of the container, as a percentage of the main column.
+$center-width-xlarge: (6/6) * 100%;
 
 .figure--full-width {
```
Comment (jamesgorrie) [inline client/scss/components/_figure.scss]: We'll need to pay heed to this [here](https://github.com/wellcometrust/wellcomecollection.org/pull/285/files#diff-28980bbc8de92d1bd1580cda008f8b1aR51) - but hopefully we can get one class to use for both.

File: server/views/pages/article.njk [hunk 2]
```diff
@@ -12,9 +12,9 @@
 
 <div class="row row--cream">
   <div class="container">
-    <div class="grid">
-      <div class="grid__cell grid__cell--xl6 grid__cell--shift-xl1 grid__cell--l7 grid__cell--m6 grid__cell--shift-m1 grid__cell--s4">
-
+    <div class="grid grid--no-gutters">
```
Comment (jamesgorrie) [inline server/views/pages/article.njk]: Does this need updating in the prod article?

File: client/scss/grids/_grid_no-gutters.scss [hunk 3]
```diff
@@ -0,0 +1,7 @@
+.grid--no-gutters {
```
Comment (jamesgorrie) [inline client/scss/grids/_grid_no-gutters.scss]: Why is this not just in the `grid` file?

File: client/scss/application.scss [hunk 4]
```diff
@@ -20,6 +20,7 @@
 @import 'grids/grid_breakout';
 @import 'grids/grid_dividers';
 @import 'grids/grid_size';
+@import 'grids/grid_no-gutters';
```
Comment (jamesgorrie) [inline client/scss/application.scss]: if we keep this files - could we use `_` instead of `-`?

File: server/views/pages/article.njk [hunk 5]
```diff
@@ -12,9 +12,9 @@
...
+      <div class="grid__cell grid__cell--s4 grid__cell--m6 grid__cell--shift-m1 grid__cell--l7 grid__cell--xl6 grid__cell--shift-xl1">
+
```
Comment (jamesgorrie) [inline server/views/pages/article.njk]: Extra white space...

File: server/views/pages/article.njk (hunk 2 above)
Comment (gestchild) [inline server/views/pages/article.njk]: Not sure I understand. This is the prod article and it's inherited by the template article.

Comment (jamesgorrie) [inline server/views/pages/article.njk]: Sorry, I completely missed that... my bad.

**Pull request merged**